<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>443</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0" columnstretch="0,0">
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="Canvas"/>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,10,10,10,10,10">
      <item>
       <widget class="QCheckBox" name="PaintModeCheckBox">
        <property name="text">
         <string>3D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LineSegmentButton">
        <property name="text">
         <string>LineSegment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CircleButton">
        <property name="text">
         <string>Circle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="EllipseButton">
        <property name="text">
         <string>Ellipse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ColorButton">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="EraserButton">
        <property name="text">
         <string>Eraser</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="ModeLine">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
